types.go

```diff
@@ -382,3 +382,29 @@ type Element struct {
 	// Comment is an optional comment for the element
 	Comment *string
 }
+
+type FlowtableHookPriority string
+
+const (
+	// FilterHookPriority stands for 0
+	FilterHookPriority FlowtableHookPriority = "filter"
```

Comment:
I wouldn't include "Hook" in the name here. (We don't for `BaseChainPriority`...)

Admittedly, the names of the `BaseChainPriority` values kind of assume that there aren't going to be any other priority types to conflict with them...

Meh. Maybe we should rename `BaseChainPriority` to something more generic and use it here?

Reply:
I would not couple the priorities unless we have guarantees netfilter is going to keep them aligned ... better to be safe than sorry, we can renames our constant but if the constant values of the backend change or are different for flowtables and chains we have a problem